File: Leetcode/res/Find the Duplicate Number/2.cpp
\*
Author: allannozomu
Runtime: 568 ms
Memory: 10.1 MB*\

class Solution {
public:
    int findDuplicate(vector<int>& nums) {
        for (int i = 0 ; i < nums.size(); ++i){
            for (int j = i + 1; j < nums.size(); ++j){
                if (nums[i] == nums[j]) return nums[i];
            }
        }
        return -1;
    }
    
};

File: Leetcode/res/Regular Expression Matching/2.py
# Author: allannozomu
# Runtime: 76 ms
# Memory: 13.4 MB

class Solution:
    def isMatch(self, s: str, p: str) -> bool:
        program = re.compile('^{}$'.format(p))
        if program.match(s):
            return True
        return False